packages/amplify_api_plugin_interface/lib/src/GraphQL/GraphQLRequest.dart

```diff
@@ -18,22 +18,34 @@ import 'package:flutter/foundation.dart';
 import '../UUID.dart';
 
 class GraphQLRequest<T> {
+  String apiName;
   String document;
   Map<String, dynamic> variables = {};
   String cancelToken = UUID.getUUID();
 
-  GraphQLRequest({@required this.document, Map<String, dynamic> variables}) {
+  GraphQLRequest(
+      {String apiName,
+      @required this.document,
+      Map<String, dynamic> variables}) {
     if (variables != null) {
       this.variables = variables;
     }
+
+    if (apiName != null) {
```

Comment:
Is this null check necessary?  We made the check above because variables = {}; but here apiName is null to begin with.